### PR TITLE
[DataFactory] Enable Avro Dataset in Public Swagger

### PR DESCRIPTION
**Pull request by yuzhangyi**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by michaeljqzq** [comment]
Please provide example including the newly added property

**Review by michaeljqzq — CHANGES_REQUESTED** [review]
Please add example

**Comment by yuzhangyi** [comment]
> Please add example
Hi, after having sync with our engineer team, we don't need external example to validate it, because we already has example at "datafactory\resource-manager\Microsoft.DataFactory\stable\2018-06-01\examples"

**Comment by yuzhangyi** [comment]
> Please add example

Hi, after having sync with our engineer team, we don't need external example to validate it, because we already has example at "datafactory\resource-manager\Microsoft.DataFactory\stable\2018-06-01\examples"

**Comment by yuzhangyi** [comment]
> Please provide example including the newly added property

Hi, after having sync with our engineer team, we don't need external example to validate it, because we already has example at "datafactory\resource-manager\Microsoft.DataFactory\stable\2018-06-01\examples"

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 1]
```diff
@@ -657,6 +657,40 @@
         "fileExtension"
       ]
     },
+    "AvroWriteSettings": {
+      "description": "Delimited text write settings.",
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Fix the comment, should say Avro

**Review by hvermis — APPROVED** [review]
Make sure to fix the description

**Comment by hvermis** [comment]
@KrisBash Could you review this one too? This is another dataset type with settings, again similar to what you already reviewed.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json (hunk 1 above)
Comment (yuzhangyi) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Thanks, will fix it later.

**Comment by yuzhangyi** [comment]
> @KrisBash Could you review this one too? This is another dataset type with settings, again similar to what you already reviewed.

Hi, @KrisBash, can you help finish reviewing this change and merge it into master? thanks!

**Comment by michaeljqzq** [comment]
@yuzhangyi , is this PR still valid? If yes, please fix the comments. Otherwise I will close it

**Comment by yuzhangyi** [comment]
> @yuzhangyi , is this PR still valid? If yes, please fix the comments. Otherwise I will close it

Hi, @michaeljqzq, this public swagger change is valid, can you help signoff and merge it into master? For the comment raised by hvermis, I will fix it in next change.

**Comment by michaeljqzq** [comment]
@yuzhangyi , we can not merge a PR in a mid-state. Please either update this one or create another PR

**Comment by yuzhangyi** [comment]
Close it and use new one.

**Pull request closed without merging**